index.htm

```diff
@@ -1,6 +1,6 @@
 ---
 title: "Gridcoin - Rewarding Scientific Distributed Computing"
-description: "Gridcoin is a cryptocurrency which rewards volunteer distributed computation performed on the BOINC platform on top of Proof of Stake."

```

Comment:
This is a major change which warrants discussion IMO

Reply:
I could potentially push this out into a separate PR if that would be better

Reply:
Actually noticed that there's more places I would have to change this, so I'll just push this out to a separate PR